README.md

```diff
@@ -1,2 +1,2 @@
 # MagicalCrypticCatalog
-This file is an Xcode playground file that describes three fun-loving cryptids!
+This is an Xcode playground file that describes three fun-loving cryptids!
```

Comment:
You did not need to zip the Playground; you could have saved it in the Git repository. It is not a binary file.

Reply:
Thanks for the feedback! I did use multiple lines of code, is it possible that the formatting got lost somewhere along the way? Speaking of which, I didn't mean to make it into a zip file, I'll try not to make that happen next time. Good call on the future proofing, I didn't even think about that.